### DISCUSSION TITLE
Do you plan to support Rust?

### DISCUSSION THREAD

**Discussion by mpiorowski** [highlight/highlight#6092] [Q&A] [answered]
We would love to use Your framework, cos of amazing Svelte integration, but we are using Rust for backend. Wonder if You plan to support it also.

**Comment by Vadman97** [comment, marked as answer]
Hi. Yes, we do! As our SDKs are build on top of opentelemetry, we would be wrapping [that SDK](https://opentelemetry.io/docs/instrumentation/rust/) with our configuration. However, using the rust opentelemetry SDK pointed to our otel ingest endpoint as is would work. We have some docs on setting that up here: https://www.highlight.io/docs/general/company/open-source/contributing/adding-an-sdk

As for our wrapper SDK, expect that to be shipped in the next quarter or so (October -> December 2023).

**Reply by Vadman97** [reply]
Rust is now officially supported via a wrapper SDK: https://www.highlight.io/docs/getting-started/backend-tracing/rust/overview
